business/mesh.go

```diff
@@ -249,6 +250,18 @@ func (in *MeshService) ResolveKialiControlPlaneCluster(r *http.Request) (*Cluste
 	return kialiControlPlaneCluster, nil
 }
 
+// convertSvcToKialiInstance converts a svc Service data structure of the
+// Kubernetes client to a KialiInstance data structure.
+func convertSvcToKialiInstance(svc *core_v1.Service) KialiInstance {
```

Comment:
This is only for Kiali services, right?  It's not  a generic find me the Kiali instance that is managing some random service?  IF so I think it would be more clear to rename this, `kialiServiceToInstance()` or something like that.  Because then it's mpre clear that, for example, the app.kubernetes.io/version label is reliable.  Because for random istio services it could very well just be `version`.

Reply:
I renamed to `convertKialiServiceToInstance`.